CoffeeChat/Cells/SchedulingPlaceCollectionViewCell.swift

```diff
@@ -10,48 +10,45 @@ import UIKit
 
 class SchedulingPlaceCollectionViewCell: UICollectionViewCell {
 
-    // MARK: Private view vars
     private let titleLabel = UILabel()
 
+    override var isSelected: Bool {
+        didSet {
+            contentView.backgroundColor = isSelected ? .pearGreen : .backgroundWhite
+        }
+    }
+
     override init(frame: CGRect) {
         super.init(frame: frame)
         contentView.backgroundColor = .clear
         self.layer.masksToBounds = false
 
+        contentView.backgroundColor = .backgroundWhite
```

Comment:
why is the contentView's background color set twice (to clear and then backgroundWhite)?

Reply:
oh good catch